apache/superset
# feat: Add report format options to Charts report modal

### SUMMARY
Report format options added to Charts report modal.

### SCREENSHOT
<img width="489" alt="betoComponent" src="https://user-images.githubusercontent.com/55605634/127610341-ea03ec10-b060-4df9-b83b-cf5f23af7c95.png">

### TESTING INSTRUCTIONS
- Click "Charts" in the upper navigation bar
- Navigate to any Chart
- Click the Calendar icon button
	- Located at the top of the chart between the timer and the "Copy chart URL to clipboard" button
- Observe the "Message Content" section of the email report modal

### ADDITIONAL INFORMATION
